libcontainer/cgroups/utils.go

```diff
@@ -479,6 +479,26 @@ func RemovePaths(paths map[string]string) (err error) {
 	return fmt.Errorf("Failed to remove paths: %v", paths)
 }
 
+// RemovePathUnified aims to remove cgroup path recursively
+// Because there may be subcgroups in it.
+func RemovePathUnified(path string) error {
```

Comment:
Since we're only using this function from `fs2`, why don't we move it to fs2 and make it private.

Maybe rename to `removeCgroupPath` or so.